### DISCUSSION TITLE
PDFs larger than 5MB dont work

### DISCUSSION THREAD

**Discussion by Twantie** [requarks/wiki#5405] [Help / Questions]
Hello,
I want to include PDFs larger than 5MB to a wikipage as a hyperlink (so the usual "upload asset and include it"), but pdfs larger than 5MB dont work, smaller ones do. A click on the hyperlink just opens an empty browser tab. I have changed the settings in the security section so it should accept files up to 50MB, but somehow it doesnt work.

-------------------------------------
OS: Debian 11 & Docker
WikiJs version: 2.5.281
Database: MySQL 8-Debian Docker Image

**Comment by NGPixel** [comment]
Are you using a reverse proxy such as nginx or cloudflare?

**Reply by Twantie** [reply]
No, no proxy. Its just the container and the according database (also ina container) and nothing more
